locales/lt-LT/messages.po

```diff
@@ -312,8 +363,12 @@ msgstr "CSV failas"
 msgid "Current value:"
 msgstr "Dabartinė reikšmė:"
 
-msgid "Currently conversion from MathML to LaTeX is not available. Editing MathML here will have the LaTex code discarded."
-msgstr "Šiuo metu konversija iš MathML į LaTeX nėra galima. Redaguojant MathML čia, LaTeX kodas bus atmestas."
+msgid ""
```

Comment:
Why you have so many initial additional lines?

Reply:
fixed